app/components/questionnaire/int-field.js

```diff
@@ -0,0 +1,10 @@
+import AnswerableField from './answerable-field';
+
+const IntField = AnswerableField.extend({
+  /**
+   * Encapsulates a numeric field for a single integer
+   * JS just uses the base AnswerableField
+   */
+});
```

Comment:
Is there any validation/error message for entering a non-integer into this field?

Reply:
Ember renders a `<input type="numeric" ...` type under the hood so you can't type non-numeric values, but I should see if it will let you enter a decimal.